backend/tracim_backend/tests/functional/test_live_messages.py

```diff
@@ -168,6 +187,71 @@ def test_api__user_live_messages_endpoint_with_GRIP_proxy__ok__nominal_case(
         assert json.loads(event.data) == {"test_message": "example"}
         assert event.event == "message"
 
+    @pytest.mark.pushpin
+    def test_api__user_live_messages_endpoint_with_GRIP_proxy__err__too_many_online_users(
+        self, pushpin, app_config: CFG, bob_user: User, admin_user: User, session,
+    ):
+        # NOTE - RJ - 07-078-2021
+        # This test does the following:
+        # - run the user connection state monitor daemon using config tests_config_user_status_connection_monitor.ini
+        # - waits for it to be connected to Pushpin
+        # - connects to the admin live messages
+        # - checks that it works
+        # - connects to the bob live messages
+        # - checks that it does not work (number of simultaneous users = 1)
+        # - disconnects from the admin live messages
+        # - connects to the bob live messages
+        # - checks that it works
+
+        env = {"TRACIM_CONF_PATH": "tests_config_user_status_connection_monitor.ini"}
+        env.update(os.environ)
+
+        with subprocess.Popen(
+            ["python3", "daemons/user_connection_state_monitor.py"], env=env, stderr=subprocess.PIPE
+        ) as monitor:
```

Comment:
The classical way would have been to create a fixture to manage this process.
Also starting a file with a relative path won't work if pytest is not started from the "right" directory.

Reply:
And `sys.executable` would give you the currently used python, instead of betting that `python3` is the right one :-)

Reply:
Fixed